packages/core/src/components/drawer/drawer.tsx

```diff
@@ -57,6 +57,14 @@ export interface IDrawerProps extends IOverlayableProps, IBackdropProps, IProps
      */
     position?: Position;
 
+    /**
+     * Whether the application should return focus to the last active element in the
+     * document after this drawer closes.
+     *
+     * @default true
+     */
+    returnFocusOnClose?: boolean;
```

Comment:
"return" sounds like an action / function. the name should probably be `shouldReturnFocusOnClose`

Reply:
First thing that comes to mind for me, since it uses `document.activeElement`, is `focusPreviouslyActiveElementOnClose` but that's pretty damn wordy.